project_name/settings/base.py

```diff
@@ -206,30 +236,15 @@
     # where the documents should use wagtail-storages.
     AWS_DEFAULT_ACL = "private"
 
+    # Additional S3 object parameters
+    AWS_S3_OBJECT_PARAMETERS = {
```

Comment:
@mebdivio why are these changes needed?

Reply:
In an ideal world, we'd be selective about which files are accessible or not, as Wagtail recommends. But for parity with the existing Fly.io deployment, I think it's fine to be public by default.